docs/design/designOverview.puml

```diff
@@ -12,6 +17,10 @@ participant "addon-datastore-api" as api
 entity nvaccessServer as server
 actor NVDA
 
+/'
+Relationships, formatted as follows:
+entityNickname -> entityNickname : label for relationship
```

Comment:
Point out the direction of the arrow, and the arrow can be two directional. Note that  the arrow will be read as "dash greater" or "less dash" or "less dash greater". It might be worth pointing out these are arrows?